gateway/ln-gateway/tests/integration_tests.rs

```diff
@@ -10,6 +10,22 @@
 //! * `IFederationApi` - fake implementation of `IFederationApi` that simulates
 //!   gateway federation client dependency.
 
+use fedimint_testing::federation::FederationTest;
+use fedimint_testing::fixtures::Fixtures;
+use fedimint_testing::gateway::GatewayTest;
+use ln_gateway::rpc::rpc_client::RpcClient;
+
+async fn fixtures() -> (FederationTest, FederationTest, GatewayTest, RpcClient) {
```

Comment:
Maybe we can rename `RpcClient`? Maybe `GatewayRpcClient`?

Reply:
Will rename in follow-up and also put password inside it.